### PR TITLE
WIP - New main menu

### PR DESCRIPTION
**Pull request by nuez**
This is a Work In Progress PR.

See #51 

This is still pending #56 

You also need to add a main menu item manually.

Also installed menu_link_attributes for adding classes to the menu items to identify them for icons.

### CODE REVIEW

**Comment by juampynr** [comment]
@nuez, how can I test this locally? Can you write a list of steps?

**Comment by nuez** [comment]
this is not ready for merging btw, but good enough for testing, although icons and BG are currently missing due to #56

1. Pull and drush cim sync
2. Cache rebuild
3. Create a menu item in the main menu (can be anything, link to <front> for example

**Comment by nuez** [comment]
4. Goto a page that is not <front> which will still be loading the landing page

File: composer.json [hunk 1]
```diff
@@ -12,19 +12,20 @@
...
     "composer/installers": "^1.0.20",
     "drupal-composer/drupal-scaffold": "^2.0.1",
     "cweagans/composer-patches": "~1.0",
-    "drupal/core": "~8.0",
-    "drush/drush": "~8.0",
-    "drupal/mailchimp": "^8.1",
-    "drupal/features": "~8.3",
-    "drupal/google_analytics": "~8.2"
-   },
+    "drupal/core": "8.*",
```
Comment (juampynr) [inline composer.json]: I am not sure if this will bite us on a prod deploy. Let's see how it goes.

**Comment by juampynr** [comment]
I am not seeing it, but don't worry, I will wait.

**Comment by nuez** [comment]
@juampynr menu is finished, please check it out.

Classes need to be added to the menu items, for the icons to show:

Let me know if you have any questions

          &.agenda:before{
            content: '\E813';
          }

          &.business-day:before{
            content: '\E815';
          }

          &.survival-guide:before{
            content: '\E803';
          }

          &.speakers:before{
            content: '\E817';
          }

          &.sponsors:before{
            content: '\E812';
          }

          &.attendees:before{
            content: '\E814';
          }

          &.news:before{
            content: '\E816';
          }

          &.sprints:before{
            content: '\E802';
          }

**Comment by juampynr** [comment]
@nuez, can you add this repository as a new remote? That way, you don't need to create PRs from your fork, which makes reviewing easier as I don't need to fetch from it either.

```
git remote set-url origin git@github.com:AsociacionDrupalES/DrupalCampSpain.git
```

**Comment by juampynr** [comment]
Works! Merging.

Today I will set up site aliases with @NITEMAN so we can download the production database (where we have the survival guide). Let's get the menu items into the menu.

@nuez, what do you suggest to do with the homepage? Are we creating a new one or updating the existing one?

**Comment by juampynr** [comment]
image

image

**Comment by nuez** [comment]
@juampynr not sure what the best approach is for the home page. I think create a new one, as the current one is specific for landing pages.

We could use the paragraphs module, or maybe a more code based approach (would be easier for deployment)

I've opened a new issue to discuss this: #66

**Comment by juampynr** [comment]
Yep, let's continue there.

**Pull request merged**